app/Models/Product.php

```diff
@@ -23,4 +26,8 @@ public function category()
         return $this->belongsTo(Category::class);
     }
 
+    public function profileProduct()
```

Comment:
```suggestion
    public function profileImage()
```
Cambialo a profileImage mejor porque hace mas referencia a que es la imagen del producto.